src/egon/data/datasets/heat_supply/individual_heating.py

```diff
@@ -612,6 +611,12 @@ def cascade_heat_supply_indiv(scenario, distribution_level, plotting=True):
             columns=["estimated_flh", "priority"],
             data={"estimated_flh": [4000, 8000], "priority": [2, 1]},
         )
+    elif scenario == "eGon100RE":
+        technologies = pd.DataFrame(
+            index=["heat_pump", "gas_boiler"],
+            columns=["estimated_flh", "priority"],
+            data={"estimated_flh": [4000, 8000], "priority": [2, 1]},
+        )
```

Comment:
In eGon100RE there are no gas boilers for individual heating. There will be only heat pumps. 
And they are already added before pypsaeur is running (as far as I know). Did you check if there is already data for individual heat pumps somewhere?

Reply:
I had a look at the function that defines the minimum heat pump capacity for pypsaeur. It does not write any data in the heat-supply tables. So it is needed to do it here. But there are no gas boilers, so they should be removed from the list.

Reply:
Comments applied in c2ceec336c39713da15f2e82456c4c37d1d573c3 and 5649fa8f1fe07ec9c5314d514aeffccb3fe8e6bd